### PR TITLE
Fix profile picture preview

### PR DESCRIPTION
**Pull request by c960657**
On `/settings/profile` the JavaScript for showing a preview of the selected profile picture prior to upload is broken.

This PR fixes the script, combines the code for avatar and account header images, makes the empty preview fields visible even when no image is currently saved in the database, and ensures that the avatar preview is cropped to a square.

### CODE REVIEW

**Review by ClearlyClaire — APPROVED** [review]
Looks good to me, although the header currently being a 1×1 white pixel makes the preview look weird:
image

**Review by ClearlyClaire — APPROVED** [review]

File: app/javascript/styles/mastodon/forms.scss [hunk 1]
```diff
@@ -309,9 +309,19 @@ code {
       border-radius: 4px;
       background: url('images/void.png');
 
+      &[src$='missing.png'] {
```
Comment (mjankowski) [inline app/javascript/styles/mastodon/forms.scss:312]: I realize this is a year old, but -- was the intention here to always hide the profile avatar image when a person has not set one yet?

Comment (c960657) [reply app/javascript/styles/mastodon/forms.scss:312]: I must admit that I have no recollection of this (I can hardly remember what I had for dinner yesterday 🤪), but AFAICT this CSS is just preserving status quo? Previously the default image was inside the `if @account.header.present?` clause, so it was also hidden before, wasn't it?

Comment (mjankowski) [reply app/javascript/styles/mastodon/forms.scss:312]: Thanks, makes sense.

I see now we need to keep some markup there for the JS hook to show preview on file presence, but don't want the missing image to be visible.

Was doing something else with responsive changes and this wasn't clear at first.

**Pull request merged**